blocks/tabs/tabs.css

```diff
@@ -0,0 +1,125 @@
+/* Tabs */
+.block.tabs .tab-controls {
+  display: flex;
+  justify-content: center;
+  margin-bottom: 20px;
+  width: 100%;
+  padding-top: 0;
+  padding-bottom: 0;
+  box-sizing: border-box;
+}
+
+.block.tabs .dropdown-toggle {
+  margin: 0;
+  border: none;
+  border-radius: 0;
+  color: white;
+  background: var(--color-purple);
+  text-transform: none;
+  font-size: 1.6rem;
+  width: 100%;
+  display: flex;
+  justify-content: right;
+  gap: 20px;
+  padding: 10px 20px 10px 0;
+}
+
+.block.tabs .tab-controls button {
+  padding: 0;
+  margin: 0;
+  border: none;
+  border-radius: 0;
+  color: white;
+  text-transform: none;
+  width: 100%;
+  text-align: right;
+  display: inline-block;
+  font-size: 1.6rem;
```

Comment:
I would suggest to use the variable `--body-font-size-m`, same on other font-size properties, I think it would be better to use a variable